SPECS/mariner-release/mariner-release.spec

```diff
@@ -67,6 +67,8 @@ rm -rf $RPM_BUILD_ROOT
 %config(noreplace) /etc/issue.net
 
 %changelog
+*   Mon Aug 30 2021 Mateusz Malisz <mamalisz@microsoft.com> - 1.0-21
```

Comment:
actually...this is for the August update (not July)....AND...it looks like I failed to merge the off-cycle 1.0 change to 1.0-dev.

Reply:
d'oh!